README.md

```diff
@@ -1,10 +1,19 @@
 # Ping Integration Example Packages
 
-As the start of a new initiative called "developer experience", we are creating case-specific integration example packages. They are focused on integration and best practices, and are deployed and integrated with your PingOne tenant using Terraform from Hashicorp. This additionally demonstrates how you can manage your P1 tenant following IaC best practices.
+As the start of a new initiative called "The Developer s'Experience", we are creating a library of use-case-specific integration example packages. They are focused on integration of Ping's products and services, best practices, and are deployed and integrated with your PingOne tenant using Terraform from Hashicorp. This additionally demonstrates how you can manage your P1 tenant following IaC best practices.
 
 We are not a custom app vendor for your line of business. That is your expertise. So the example app UI, and our demo use cases are 2nd class citizens to the purpose of this project. They are merely a conduit to the goal; Ping integration examples.
 
 The sample application source code is the most basic HTML and Javascript implementation to not muddy the source with artifacts irrelevant to the package.
 
+## What's Inside
+*Check the README file in each package for more details.*
+- OIDC SDK integration.
+- DaVinci widget for PingOne user registration.
+- DaVinci API for PingOne user registration.
+- PingOne Protect and the signals SDK via DaVinci.
+- PingOne Protect using POP APIs.
+- OIDC SSO and PingOne Protect va Davinci.
```

Comment:
Missing an i in "via". Also, we should add Cayla's pkg too since its in the same sprint.